.editorconfig

```diff
@@ -0,0 +1,10 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
Hmm, if you've branched off a branch worth mentioning it in the PR title so it's super clear.